google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -1346,6 +1456,14 @@ public String getString() {
       return value.getStringValue();
     }
 
+    @Override
+    public float getFloat32() {
+      checkNotNull();
+      Preconditions.checkState(
+          value.hasNumberValue(), "This value does not contain a number value");
+      return (float) value.getNumberValue();
+    }
+
```

Comment:
This method is not needed and should be removed.

```suggestion
```

Reply:
Thanks, this seems to be a remnant of an earlier commit before GrpcResultSet